src/Uno.UI/UI/Xaml/Window/Native/NativeWindowWrapper.iOS.cs

```diff
@@ -68,7 +68,7 @@ internal void RaiseNativeSizeChanged()
 		var newWindowSize = GetWindowSize();
 
 		Bounds = new Rect(default, newWindowSize);
-		Size = newWindowSize.ToSizeInt32();
+		Size = new((int)(newWindowSize.Width * RasterizationScale), (int)(newWindowSize.Height * RasterizationScale));
```

Comment:
How about macOS `src/Uno.UI/UI/Xaml/Window/Native/NativeWindowWrapper.macOS.cs` ?

Reply:
`netx.0-macOS` is not used anymore (catalyst and desktop are favored), we'll likely remove it in the next major.